### DISCUSSION TITLE
Adding Icon to Channel

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1251] [Q&A] [answered]
My requirement is to have a logo per channel. When I display the name of the Channel, I should also display the logo that is associated with it. I would like to upload the logo into the Asset manager and link it during Channel creation. I would be using the mutation to do that. Please guide how can I extend the Channel Entity and link the asset id for the icon attribute.

**Comment by michaelbromley** [comment, marked as answer]
Hi,

To do this you can define a [custom field](https://www.vendure.io/docs/developer-guide/customizing-models/) on the Channel entity, with the type "relation" and relate it to the Asset entity.

There's an example here: https://www.vendure.io/docs/developer-guide/customizing-models/#relations

**Reply by ramkishore-yerubandi** [reply]
Thanks
